Lesson_3_HW/src/main/java/GU_Android_1089/Lesson_3/HW/Homework_Practice_GuessWordGame.java

```diff
@@ -0,0 +1,89 @@
+package GU_Android_1089.Lesson_3.HW;
+
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Homework_Practice_GuessWordGame {
+
+    public static void main(String[] args) {
+
+        init();
+
+    }
+
+    private static void init(){
+        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado", "broccoli", "carrot", "cherry",
+                "garlic", "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", "pea", "peanut",
+                "pear", "pepper", "pineapple", "pumpkin", "potato"};
+        int hiddenWordIndex = genIntValue();
+        String hiddenWord = words[hiddenWordIndex];
+        String userWord = getUserWord();
+        CompareWords(hiddenWord, userWord);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.